### PR TITLE
Tweak CMake config setup

### PR DESCRIPTION
**Pull request by davidbeckingsale**
This PR modifies RAJA-config.cmake to find camp. It also removes a lot of options we were re-defining, but the list was out of date.

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
This still needs a little debugging (I'm doing that in conjunction with the project template)...

**Comment by trws** [comment]
This looks all good to me, also agree it should take care of #663, or at least the requirement for the camp addition there, I don't recall if there was something else there that we still need.

**Comment by davidbeckingsale** [comment]
Yeah that was just adjusting the documentation, but with this fix, that's not required 😄

**Comment by davidbeckingsale** [comment]
@rhornung67 - docs updated.

File: docs/sphinx/user_guide/using_raja.rst [hunk 1]
```diff
@@ -32,23 +32,10 @@ Then, pass the path of RAJA to CMake when you configure your code::
...
-``RAJA_INCLUDE_DIR``     Include directory for RAJA headers.
-``RAJA_LIB_DIR``         Library directory for RAJA.
-``RAJA_COMPILE_FLAGS``   C++ flags used to compile RAJA.
-``RAJA_NVCC_FLAGS``      CUDA flags used to compile RAJA.
-======================   ===================================
-
-It also provides the ``RAJA`` target, that can be used natively by CMake to add
-a dependency on RAJA. For example::
+The ``RAJA-config.cmake`` file provides a ``RAJA`` target, that can be used
+natively by CMake to add a dependency on RAJA. For example::
```
Comment (rhornung67) [inline docs/sphinx/user_guide/using_raja.rst]: I think you deleted too much here. We need some or all of that sentence after the table to explain the two CMakeList.txt file lines.

Comment (davidbeckingsale) [reply docs/sphinx/user_guide/using_raja.rst]: I don't think so. The stuff I deleted no longer exists.

How do you think it should read?

Comment (rhornung67) [reply docs/sphinx/user_guide/using_raja.rst]: Nevermind. I see what you did now. I was confused before.

**Pull request merged**